modules/strings/strings.go

```diff
@@ -15,63 +18,29 @@ func asString(obj object.Object) (*object.String, *object.Error) {
 	return s, nil
 }
 
-func Contains(ctx context.Context, args ...object.Object) object.Object {
-	if err := arg.Require("strings.contains", 2, args); err != nil {
-		return err
-	}
-	s, err := asString(args[0])
-	if err != nil {
-		return err
-	}
-	return s.Contains(args[1])
+//risor:export
+func Contains(s, substr string) bool {
+	return strings.Contains(s, substr)
```

Comment:
Would it be possible to use this form?

```
var Contains = strings.Contains
```

Presumably the Go AST is different, but maybe this case could be handled?

Reply:
Yeah the AST doesn't contain type information.

The approach would be to use [`go/types`](https://pkg.go.dev/go/types) to parse the type info, and potentially combine that with [`go/importer`](https://pkg.go.dev/go/importer). And then from that resolve the function definition of `strings.Contains`.

Certainly doable, but I'd say better to leave that for a follow-up PR.